zerver/lib/events.py

```diff
@@ -397,6 +397,10 @@ def fetch_initial_state_data(
         ] = settings.TYPING_STARTED_WAIT_PERIOD_MILLISECONDS
 
         state["server_supported_permission_settings"] = get_server_supported_permission_settings()
+
+        state[
+            "can_access_all_users_group_limits_presence"
```

Comment:
Still need to add API documentation about this.

Reply:
https://github.com/zulip/zulip/pull/27944/commits/6f613daabf7ffcfd1df525d0a7efaffd411e98da#r1408603397 probably means you don't need this in the API.